go/client/cmd_upak.go

```diff
@@ -44,11 +49,23 @@ func (c *cmdUPAK) Run() error {
 	if err != nil {
 		return err
 	}
-
 	if err = RegisterProtocolsWithContext(nil, c.G()); err != nil {
 		return err
 	}
 
+	if c.lite {
+		upakLite, err := userClient.GetUPAKLite(context.Background(), c.uid)
+		if err != nil {
+			return err
+		}
+		jsonOut, err := json.Marshal(upakLite)
```

Comment:
MarshalIndent might be nicer for debugging

Reply:
cool. i'm not super opinionated; i prefer to pipe it to jq either way.